packages/runtime/container-runtime/src/test/dataStoreContext.spec.ts

```diff
@@ -333,94 +339,80 @@ describe("Data Store Context Tests", () => {
-                };
-                const buffer = stringToBuffer(JSON.stringify(dataStoreAttributes), "utf8");
-                const blobCache = new Map<string, ArrayBufferLike>([["fluidDataStoreAttributes", buffer]]);
-                const snapshotTree: ISnapshotTree = {
-                    blobs: { [dataStoreAttributesBlobName]: "fluidDataStoreAttributes" },
-                    commits: {},
-                    trees: {},
-                };
-
-                remotedDataStoreContext = new RemotedFluidDataStoreContext(
-                    dataStoreId,
-                    snapshotTree,
-                    containerRuntime,
-                    new BlobCacheStorageService(storage as IDocumentStorageService, blobCache),
-                    scope,
-                    createSummarizerNodeFn,
-                );
-
-                const isRootNode = await remotedDataStoreContext.isRoot();
-                assert.strictEqual(isRootNode, true, "The data store should be root.");
-
-                remotedDataStoreContext.updateUsedRoutes([""]);
-
-                const summarizeResult = await remotedDataStoreContext.summarize(true /* fullTree */);
-                assert(summarizeResult.summary.type === SummaryType.Tree,
-                    "summarize should always return a tree when fullTree is true");
-                const blob = summarizeResult.summary.tree[dataStoreAttributesBlobName] as ISummaryBlob;
-
-                const contents = JSON.parse(blob.content as string) as IFluidDataStoreAttributes;
-                assert.strictEqual(contents.pkg, dataStoreAttributes.pkg, "Remote DataStore package does not match.");
-                assert.strictEqual(
-                    contents.snapshotFormatVersion,
-                    dataStoreAttributes.snapshotFormatVersion,
-                    "Remote DataStore snapshot version does not match.");
-                assert.strictEqual(
-                    contents.isRootDataStore,
-                    dataStoreAttributes.isRootDataStore,
-                    "Remote DataStore root state does not match");
-            });
+                }));
 
-            it("can correctly initialize and generate attributes without version and isRootDataStore", async () => {
-                dataStoreAttributes = {
-                    pkg: "TestDataStore1",
-                    snapshotFormatVersion: undefined,
-                };
-                const buffer = stringToBuffer(JSON.stringify(dataStoreAttributes), "utf8");
-                const blobCache = new Map<string, ArrayBufferLike>([["fluidDataStoreAttributes", buffer]]);
-                const snapshotTree: ISnapshotTree = {
-                    blobs: { [dataStoreAttributesBlobName]: "fluidDataStoreAttributes" },
-                    commits: {},
-                    trees: {},
-                };
+                it("reading from latest without isolated channels", async () => testGenerateAttributesCore({
```

Comment:
Can you add another test case with `disableIsolatedChannels: false`. I understand that this attributes not being there means its false. But since this is a boolean and can accept false, we should have a test case to make sure we don't break this scenario.

Reply:
It is typed as `true | undefined`.

Reply:
oh okay! I looked at the one in IContainerRuntimeOptions and got confused :-).